src/pages/ConfirmationPage.js

```diff
@@ -70,20 +74,33 @@ class ConfirmationPage extends React.Component {
     }
   }
 
-  hashFromData(email, paperFileNumber) {
-    var hash = 0,
-      i,
-      chr
-    const keys = email + paperFileNumber
-    if (keys.length === 0) return hash
-    for (i = 0; i < keys.length; i++) {
-      chr = keys.charCodeAt(i)
-      hash = (hash << 5) - hash + chr
-      hash |= 0
-    }
-    return hash
+  getEmailNum(documentId) {
+    axios
+      .get(`/appointments/confirm/${documentId}`)
+      .then(locs => {
+        this.setState({
+          comfirmNum: locs.data.confirmation,
+        })
+      })
+      .catch(function() {
+        this.props.history.push('/error')
+      })
   }
 
+  // hashFromData(email, paperFileNumber) {
+  //   var hash = 0,
+  //     i,
+  //     chr
+  //   const keys = email + paperFileNumber
+  //   if (keys.length === 0) return hash
+  //   for (i = 0; i < keys.length; i++) {
+  //     chr = keys.charCodeAt(i)
+  //     hash = (hash << 5) - hash + chr
+  //     hash |= 0
+  //   }
+  //   return hash
+  // }
```

Comment:
There shouldn't be a need to keep commented out code.

Reply:
Looks like this was removed